care/facility/api/viewsets/shifting.py

```diff
@@ -69,6 +69,7 @@ class ShiftingFilterSet(filters.FilterSet):
     assigned_to = filters.NumberFilter(field_name="assigned_to__id")
     created_by = filters.NumberFilter(field_name="created_by__id")
     last_edited_by = filters.NumberFilter(field_name="last_edited_by__id")
+    is_antenatal = filters.BooleanFilter(field_name="patient__is_antenatal")
```

Comment:
Add a default to None , rather than false , also create the required migration files, add a filter to the list api as well

Reply:
Can you please specify which list API you are referring to? Thanks.

Reply:
The shifting list API